tests/user/test_user_update.py

```diff
@@ -0,0 +1,320 @@
+from allure import feature, description, step, title
+from hamcrest import assert_that, is_
+
+from framework.asserts.common import (
+    assert_status_code,
+    assert_content_type,
+    assert_response_message,
+)
+from framework.asserts.user_asserts import assert_update_user_data_matches
+from framework.endpoints.users_api import UsersAPI
+from framework.tools.generators import generate_user
+import pytest
+
+
+@feature("Update user's information")
+class TestUpdateUser:
+    @title("Updating User Information")
+    @description(
+        "GIVEN the user is logged in, "
+        "WHEN the user sends a request to update their own information, "
+        "THEN the response code is 200 and the response body contains the current user's data."
+    )
+    def test_update_user_info(self, create_authorized_user):
+        token = create_authorized_user["token"]
```

Comment:
Can I ask you to wrap in a `step`?

Reply:
Thanks for the review and the idea to use an Allure step for the line token = create_authorized_user["token"]].

I think this line is just a simple setup step and not a major part of the test. Adding it as an Allure step might make our reports too detailed and less clear. What's your opinion on this?

Please also keep in mind that create_authorized_user["token"]] has already 2 allure steps

```
@title("Creating an authorized user")
@fixture(scope="function")
def create_authorized_user(postgres):
  with step("Creating user in DB"):
          user_to_create = generate_and_insert_user(postgres)
  
  with step("Authentication of user and getting token"):
          authentication_response = AuthenticateAPI().authentication(
              email=user_to_create["email"], password=user_to_create["password"]
          )
          token = authentication_response.json()["token"]
  
  yield {"user": user_to_create, "token": token}
```